### PR TITLE
Dash setup initial testing 54

### PR DESCRIPTION
**Pull request by joshuawootonn**
### Completed: 
* Testing setup
* Testing above 80% (82.04%)
* Remove Live Stream features, but keep the code, until we actually use it

### New Things To Do: 
* Refactor to TS and test the MapView Component

### Coverage
image

### CODE REVIEW

**Comment by jwryan9** [comment]
@hhpatel09 can you make sure these tests get run in the CI with every build?

File: Dashboard/src/components/EStop.tsx [hunk 1]
```diff
@@ -4,19 +4,9 @@ import "../css/EStop.css";
...
-      })
-      .catch(err => {
-        console.error(err);
-      });
-  }
-
   render() {
     return (
-      <a className="EStop" href="localhost://#" onClick={this.click}>
+      <a className="EStop" href="localhost://#" onClick={API.estop}>
```
Comment (jwryan9) [inline Dashboard/src/components/EStop.tsx]: Should this href be changed so we don't render a blank page on estop click? Maybe have some overlay on the dashboard saying estop has been engaged.

**Comment by joshuawootonn** [comment]
@jwryan9  Ya I agree there are a lot of <a> tags used where buttons should be. The styling of the two is slightly different since they have different display properties. I think that is why he did it.

**Comment by joshuawootonn** [comment]
I created a feature in ice box for this! @jwryan9  @cooperbell  #74

**Pull request merged**